lib/presentation/util/helper/responsive_utils.dart

```diff
@@ -86,18 +88,44 @@ class ResponsiveUtils {
 
   EdgeInsets getMarginForDestinationPicker(BuildContext context) {
     if (isMediumScreen(context)) {
-      return EdgeInsets.symmetric(
-        horizontal: _destinationPickerHorizontalMarginMediumScreen,
-        vertical: _destinationPickerVerticalMarginMediumScreen);
+      return getSizeHeightScreen(context) <= _destinationPickerVerticalMarginMediumScreen * 2
+        ? EdgeInsets.symmetric(
+            horizontal: _destinationPickerHorizontalMarginMediumScreen,
+            vertical: 0.0)
+        : EdgeInsets.symmetric(
+            horizontal: _destinationPickerHorizontalMarginMediumScreen,
+            vertical: _destinationPickerVerticalMarginMediumScreen);
     } else if (isLargeScreen(context)) {
-      return EdgeInsets.symmetric(
-        horizontal: _destinationPickerHorizontalMarginLargeScreen,
-        vertical: _destinationPickerVerticalMarginLargeScreen);
+      return getSizeHeightScreen(context) <= _destinationPickerVerticalMarginLargeScreen * 2
+        ? EdgeInsets.symmetric(
+            horizontal: _destinationPickerHorizontalMarginLargeScreen,
+            vertical: 0.0)
+        : EdgeInsets.symmetric(
+            horizontal: _destinationPickerHorizontalMarginLargeScreen,
+            vertical: _destinationPickerVerticalMarginLargeScreen);
     } else {
       return EdgeInsets.zero;
     }
   }
 
+  BorderRadius getBorderRadiusView(BuildContext context) {
+    if (isMediumScreen(context)) {
+      return getSizeHeightScreen(context) <= _destinationPickerVerticalMarginMediumScreen * 2
+        ? BorderRadius.only(
+            topLeft: Radius.circular(_radiusDestinationPickerView),
+            topRight: Radius.circular(_radiusDestinationPickerView))
+        : BorderRadius.circular(_radiusDestinationPickerView);
+    } else if (isLargeScreen(context)) {
+      return getSizeHeightScreen(context) <= _destinationPickerVerticalMarginLargeScreen * 2
```

Comment:
Can you explain a little bit?

Reply:
We check when the height of the screen is less than or equal to the distance to the margin that we have set when displaying on tablet devices, we will not be separated from the margin vertically but only horizontally.